src/utils.ts

```diff
@@ -362,17 +438,24 @@ const applyFactor = (value: BigNumberish, factor: number) => {
   );
 };
 
+const isDataEmpty = (data: string) => {
+  return ['', '0x', '0x00'].includes(data);
```

Comment:
'0x00' is also included in the check `BigNumber.from(data).isZero()`

Reply:
But here we are not checking data as a number.